### DISCUSSION TITLE
Welcome to clavascript Discussions!

### DISCUSSION THREAD

**Discussion by borkdude** [clavascript/clavascript#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by laheadle** [comment]
This is cool! Reminds me of parenscript from back in the day. I'm a fan. I'm thinking of using this to write a Chrome extension to help me automate a few personal tasks. Maybe using this framework.

https://github.com/dutiyesh/chrome-extension-cli

**Reply by borkdude** [reply]
Hey @laheadle - thanks! This looks really interesting. Keep us posted.
